src/client/l10n/locales.ts

```diff
@@ -0,0 +1,147 @@
+import { Locale as IsoLocale } from 'data/graphql'
+
+export type LocaleLabel = 'se' | 'se-en' | 'no' | 'no-en' | 'dk' | 'dk-en'
+
+export type LocaleData = {
+  path: LocaleLabel
+  isoLocale: IsoLocale
+  marketLabel: MarketLabel
+  htmlLang: 'en' | 'sv' | 'no' | 'da'
+  ssn: Ssn
+  birthdate: Birthdate
+}
+
+type MarketLabel = 'SE' | 'NO' | 'DK'
+
+type Ssn = {
+  length: number
+  formatExample: string
+  formatRegex: RegExp
+}
+
+type Birthdate = {
+  formatExample: string
+  formatRegex: RegExp
+  defaultFormatExample: string
+}
+
+const ssnFormats: Record<MarketLabel, RegExp> = {
+  SE: /^((19|20))([0-9]{2})(0[1-9]|1[0-2])(0[1-9]|[1-2][0-9]|3[0-1])([0-9]{4})$/,
+  NO: /^(0[1-9]|[1-2][0-9]|3[0-1])(0[1-9]|1[0-2])([0-9]{2})([0-9]{5})$/,
+  DK: /^(0[1-9]|[1-2][0-9]|3[0-1])(0[1-9]|1[0-2])([0-9]{2})([0-9]{4})$/,
+}
+
+const ssnLengths: Record<MarketLabel, number> = {
+  SE: 12,
+  NO: 11,
+  DK: 10,
+}
+
+const yearMonthDayFormat = /^(19[0-9]{2}|20[0-9]{2})-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])$/
+const dayMonthYearFormat = /^(0[1-9]|[1-2][0-9]|3[0-1])-(0[1-9]|1[0-2])-(19[0-9]{2}|20[0-9]{2})$/
+
+export const birthdateFormats: Record<MarketLabel | 'default', RegExp> = {
+  default: yearMonthDayFormat,
+  SE: yearMonthDayFormat,
+  NO: dayMonthYearFormat,
+  DK: dayMonthYearFormat,
+}
+
```

Comment:
Maybe all this SSN and birthdate stuff should be in its own file in the same folder? Or what do you think?